PowerFGT/Public/Connection.ps1

```diff
@@ -153,92 +151,116 @@ function Connect-FGT {
             $url = "https://${Server}:${port}/"
         }
 
-        $uri = $url + "logincheck"
-        $iwrResponse = $null
-        try {
-            $iwrResponse = Invoke-WebRequest $uri -Method POST -Body $postParams -SessionVariable FGT @invokeParams
-        }
-        catch {
-            Show-FGTException $_
-            throw "Unable to connect to FortiGate"
+        $headers = @{}
+        if ($ApiToken) {
+            $uri = $url + "logincheck"
+            try {
+                $apiLoginHeaders = @{ "Authorization" = "Bearer $ApiToken" }
+                Invoke-WebRequest $uri -Method GET -Headers $loginHeaders -SessionVariable FGT @invokeParams | Out-Null
```

Comment:
really need this WebRequest ? (when use the API)

Typo on -Headers variable name ?

Reply:
You're right, this request just fails when and is not needed.